internal/compiler/widgets/native/std-widgets.slint

```diff
@@ -120,18 +166,43 @@ export ComboBox := NativeComboBox {
             }
         }
     }
+
+    fs := FocusScope {
+        key-pressed(event) => {
+            if (event.text == Keys.UpArrow) {
+                root.current-index = Math.max(root.current-index - 1, 0);
+                root.current-value = model[root.current-index];
+                return accept;
+            } else if (event.text == Keys.DownArrow) {
+                root.current-index = Math.min(root.current-index + 1, root.model.length - 1);
+                root.current-value = model[root.current-index];
+                return accept;
+            // PopupWindow can not get hidden again at this time, so do not allow to pop that up.
+            // } else if (event.text == Keys.Return) {
+            //     touch.clicked()
+            //     return accept;
+            }
+            return reject;
+        }
+    }
 }
 
 export TabWidgetImpl := NativeTabWidget { }
 
-export TabImpl := NativeTab { }
+export TabImpl := NativeTab {
+    accessible-role: tab;
+    accessible-label <=> title;
+}
 
 export TabBarImpl := Rectangle {
     // injected properties:
     property<int> current; // The currently selected tab
     property<int> current-focused: fs.has-focus ? current : -1; // The currently focused tab
     property<int> num-tabs; // The total number of tabs
 
+    accessible-role: tab;
+    // accessible-delegate-focus: current;
```

Comment:
leftover comment

but don't it actually need to be set?

Reply:
I enable that again later. It crashed in native style when I was updating from the PropertyTracker without going through the event loop first.